inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/custom/CustomSinkRequest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.pojo.sink.custom;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.apache.inlong.manager.common.consts.SinkType;
+import org.apache.inlong.manager.common.util.JsonTypeDefine;
+import org.apache.inlong.manager.pojo.sink.SinkRequest;
+
+import javax.validation.constraints.NotBlank;
+
+/**
+ * Base sink request
+ */
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@ApiModel(value = "Custom sink request")
+@JsonTypeDefine(value = SinkType.CUSTOM)
+public class CustomSinkRequest extends SinkRequest {
+
+    @NotBlank(message = "sinkType cannot be blank")
+    @ApiModelProperty("Sink type, including: HIVE, ES, etc.")
+    private String sinkType;
```

Comment:
The parent class already has this field.

Reply:
fixed, thx